changelog/248.breaking.rst

```diff
@@ -0,0 +1,5 @@
+Rework :class:`disnake.enums.Enum` internals.
```

Comment:
This filename should be 556.

Reply:
Fixed in 0037617fe4a30dde0d20de37c956a2abd2f4fb58. Thanks for the heads-up; no idea how that happened.